src/segmentmytif/main.py

```diff
@@ -49,7 +54,7 @@ def read_input_and_labels_and_save_predictions(
     dask_kwargs = _set_compute_mode(compute_mode, chunks)
 
     # Load raster data
-    raster = rioxarray.open_rasterio(raster_path, **dask_kwargs)
+    raster = read_geotiff(raster_path)
```

Comment:
See my suggestion in `io.py`. I suggest to move `_set_compute_mode` inside `read_geotiff`.
Let's add the `compute_mode` (str) and `chunks` (dict) as the input arguments here maybe?